src/test/java/karate-config.js

```diff
@@ -1,18 +1,22 @@
 function fn() {    
-  var env = karate.env; // get system property 'karate.env'
+  var env = karate.env;
   karate.log('karate.env system property was:', env);
   if (!env) {
     env = 'dev';
   }
   var config = {
     env: env,
-	myVarName: 'someValue'
+    someUrl: 'https://angular-conduit-node.herokuapp.com/api',
+    register:{
+    		username: 'QAtomuyuaiut',
+            email: 'qaua65iony@gmail.com',
+            password: 'elvism67898'
+    	}
   }
```

Comment:
good work with that! BUT we must not commit this data to the repository. Try to move this to a _.properties_ or _.env_ or whatever and don't push it.